test/unit/README.md

```diff
@@ -0,0 +1,2 @@
+These directory contains test, that check new driver logic and are expected to pass.
```

Comment:
"contains tests" instead "contains test", "This directory" instead "These directory"